api/app/GraphQL/Mutations/CreateOrUpdateGeneralQuestionAssessmentStep.php

```diff
@@ -33,7 +33,7 @@ public function __invoke($_, array $args)
                     array_push($incomingQuestionIds, $incomingQuestion['id']);
                     $questionToUpdate = $existingQuestions->find($incomingQuestion['id']);
                     if ($questionToUpdate === null) {
-                        throw new Exception('ScreeningQuestionNotExist');
+                        throw new Exception('GeneralQuestionNotExist');
```

Comment:
Don't forget to rename the matching message in apiMessages

Reply:
I don't think that should get renamed, it would be confusing to have it with "general" in the name but the copy contains "screening". 
I think a new API message should be added with "general question" specific copy in the follow-up issue

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/73d8ea8e-e681-4799-9246-91e0d5b386e6)

Reply:
That's fair.  Could you add it as an AC item in the follow-up issue to ensure it gets tied up later?

Reply:
Updated https://github.com/GCTC-NTGC/gc-digital-talent/issues/8676